lib/phlex/view.rb

```diff
@@ -25,6 +35,10 @@ def compile
 			def compiled?
 				!!@compiled
 			end
+
+			def inherited(child)
+				child.prepend(TemplateOverride)
+			end
```

Comment:
When `Phlex::View` is inherited, we want to prepend the `TemplateOverride` module on the child so that `template` is defined by our override.